### DISCUSSION TITLE
Mentoring needed

### DISCUSSION THREAD

**Discussion by gkum99** [kedacore/keda#3767] [Q&A / Need Help]
Hi guys, i am new to opensource. I picked good first issue of Refactor MS SQL e2e test #3401 but i am unable to understand issue and fix it. I have done building keda project inside my vscode. Please help me with later steps.

**Comment by tomkerkhove** [comment]
Sure, we suggest you to join the Kubernetes Slack where we have #keda-dev for these kind of things.
See https://keda.sh/community/

Thanks for contributing!
